src/layout/section/columnedContentSection/ColumnedContentSection.tsx

```diff
@@ -106,12 +107,13 @@ const renderColumnContent = (columnContent: ColumnContent, index: string): JSX.E
   return (
     <Column className="kirk-columned-content-section-column" key={index}>
       {media && renderMedia(media)}
+      <p className="kirk-columned-content-section-subtitle">
```

Comment:
I found this a bit weird, it does look like a title.

Reply:
The change of tag is to avoid adding a cluster of headings that might bring more noise to bots than intended.

As for the look, I kept it the same indeed.